<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteCommentsDlg</class>
 <widget class="QDialog" name="RouteCommentsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="txtTags"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnPrev">
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="HtmlTextEdit" name="txtComments"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select route(s):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDateEdit" name="dateEdit">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="RouteSelector" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the route to which this comment applies. Multiple selections may be made if the same comment applies to more than one route.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Tags:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnNext">
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HtmlTextEdit</class>
   <extends>QTextEdit</extends>
   <header>htmltextedit.h</header>
  </customwidget>
  <customwidget>
   <class>RouteSelector</class>
   <extends>QTableView</extends>
   <header>routeselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
